### ISSUE TITLE
Guide on editing resource quotas

### ISSUE THREAD

**Issue by vparekh94** [astronomerio/airflow-guides#60]
As  an EE customer, I need to edit resource quotas on  my namespaces through kubectl:

**Comment by vparekh94** [comment]
Some commands:
```kubectl get resourcequota --namespace <deployment_namespace>```

**Comment by bonominijl** [comment]
@cwurtz Can this be done through Houston without it being overridden on the next deploy?

**Comment by cwurtz** [comment]
Yes, you can edit the defaults on the `deployment_constraints` table

**Comment by cwurtz** [comment]
If it needs to be done for a specific deployment, it can be done via the deployment config, or adding a new constraints.  But if they are fine with all deployments being increases, editing the system level constraint is the way to go

**Comment by vparekh94** [comment]
based on our 0.7 lauch, this can be closed
